samples/test/get_operation_status.test.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2019 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const {assert} = require('chai');
+const {AutoMlClient} = require('@google-cloud/automl').v1;
+
+const cp = require('child_process');
+
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+
+const GET_OPERATION_STATUS_REGION_TAG = 'get_operation_status';
+const LOCATION = 'us-central1';
+const OPERATION_ID = 'TRL5980949629938696192';
+
+describe('Automl Get Operation Status Tests', () => {
+  const client = new AutoMlClient();
+
+  it('should get operation status', async () => {
+    const projectId = await client.getProjectId();
+
+    const get_output = execSync(
```

Comment:
Is this just looking for a static operation that has completed? If so, will that operation ever be deleted from wherever it's stored? Otherwise, can we make an operation that we then cancel and search for right away?